File: Projeto_4/src/ContaBancaria.cpp
#include "ContaBancaria.h"

ContaBancaria::ContaBancaria(std::string titular, std::string cpf, double saldoInicial)
    : titular(titular), cpf(cpf), saldo(saldoInicial) {}

void ContaBancaria::exibirDados() const {
    std::cout << "Titular: " << this->titular 
              << " | CPF: ***.***.***-**" 
              << " | Saldo: R$ " << this->saldo << std::endl;
}

File: Projeto_4/models/ContaBancaria.py
class ContaBancaria:
    def __init__(self, titular: str, cpf: str, saldo: float):
        self.__titular = titular 
        self.__cpf = cpf          
        self._saldo = saldo       

    def exibirDaddos(self) -> None:
        print(f"Titular: {self.__titular} | Saldo Atual: R$ {self._saldo:.2f}")